build_tools/embed_data/build_defs.bzl

```diff
@@ -14,6 +14,13 @@
 
 """Embeds data files into a C or C++ module."""
 
+def clean_dep(dep):
+    """Returns an absolute Bazel path to 'dep'.
+
+    This is necessary when calling these functions from another workspace.
+    """
+    return str(Label(dep))
+
 def cc_embed_data(
```

Comment:
Thanks for the explanation. I'm happy to switch to c_embed_data for now. I assume the main difference is the lack of namespace support?
Of course I'd prefer to avoid the dependency on IREE for just one Bazel rule, and as soon as https://github.com/bazelbuild/rules_cc/issues/41 gets fixed I'd switch to that.

Reply:
Rebase done.